cdisc-org/conformance-rules-editor
# ISS 179, 174: schema updates for fda rules, send rule id

To test:
Run rule editor locally against dev rules database.
1. Navigate to `Rule Ids`=`SEND2`. Ensure there are no yellow warnings in the editor pane.
2. Navigate to any rule where `Orgs`=`FDA`. Ensure there are no yellow warnings in the editor pane.
